src/org/camunda/latera/bss/connectors/hid/Hydra.groovy

```diff
@@ -49,10 +51,10 @@ class Hydra implements Ref, Message, DataType, AddParam, Good, Document, Contrac
     this.hid        = new HID(execution)
     def ENV         = System.getenv()
 
-    this.locale     = execution.getVariable('locale')
-    this.user       = ENV['HYDRA_USER']     ?: execution.getVariable('hydraUser') ?: 'hydra'
+    this.locale     = execution.getVariable('locale') ?: DEFAULT_LOCALE
+    this.user       = ENV['HYDRA_USER']     ?: execution.getVariable('hydraUser') ?: DEFAULT_USER
     this.password   = ENV['HYDRA_PASSWORD'] ?: execution.getVariable('hydraPassword')
-    this.firmId     = toIntSafe(execution.getVariable('hydraFirmId')     ?: (execution.getVariable('homsOrderDataFirmId') ?: getDefaultFirmId()))
+    this.firmId     = toIntSafe(execution.getVariable('hydraFirmId')     ?: (execution.getVariable('homsOrderDataFirmId') ?: DEFAULT_FIRM))
```

Comment:
seems like DEFAULT_FIRM is undefined?

Reply:
No, it's imported from Constants class:
```groovy
import static org.camunda.latera.bss.utils.Constants.DEFAULT_FIRM
```

See at the top of the file.